src/libstd/primitive_docs.rs

```diff
@@ -413,6 +415,8 @@ mod prim_isize { }
 //
 /// The pointer-sized unsigned integer type.
 ///
+/// `usize` is guaranteed to be at least as large as `u16`.
```

Comment:
> don't think this change to the language (and therefore RFC) is required. If we ever support a platform with 8-bit pointers (which is unlikely, but still) it would be much easier to just insert a cfg before impl_from! { u16, usize }.

Even if we wanted to support the tiny memory model, `usize` would still have to be at least 16 bits, because `size_t` would still be 16 bits, and `usize` being smaller than `size_t` would mean that `usize` is smaller than `size_t`, which would make dealing with the FFI in a generic way very complicated.